.github/workflows/charterafrica-deploy-prod.yml

```diff
@@ -10,10 +10,9 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  DOKKU_REMOTE_BRANCH: "master"
-  DOKKU_REMOTE_URL: "ssh://dokku@charter.prod.codeforafrica.org/charterafrica"
+  DOKKU_REMOTE_URL: "ssh://dokku@ui-1.prod.codeforafrica.org/charterafrica-staging"
```

Comment:
I take it the dokku app is indeed called `charterafrica-staging`?

Reply:
Yes `charterafrica-staging`